docs/debugging-openshift.md

```diff
@@ -6,7 +6,7 @@ This document contains some tips and suggestions for troubleshooting an OpenShif
 System Environment
 ------------------
 
-1. Run as root
+1. Run as root.
```

Comment:
Thanks for the update.  It looks like this is missing from the next 2 numbered items as well.  Please update for those.